deploy/operator/mirror_utils.sh

```diff
@@ -93,6 +93,17 @@ function mirror_package_from_official_index() {
   mirror_package "${package}" "${remote_index}" "${local_registry}" "${authfile}" "${catalog_source_name}"
 }
 
+function mirror_file() {
+  remote_url="${1}"
+  httpd_path="${2}"
+  base_mirror_url="${3}"
+
+  local file_name="${remote_url##*/}"
```

Comment:
I find using `basename` to be much clearer operation
```bash
➜  ~ x="https://mirror.openshift.com/pub/openshift-v4/dependencies/rhcos/4.8/4.8.0-fc.0/rhcos-4.8.0-fc.0-x86_64-ostree.x86_64.tar"
➜  ~ echo ${x##*/}
rhcos-4.8.0-fc.0-x86_64-ostree.x86_64.tar
➜  ~ basename ${x}
rhcos-4.8.0-fc.0-x86_64-ostree.x86_64.tar
```

Reply:
done